sql/src/main/java/io/crate/analyze/SetStatementAnalyzer.java

```diff
@@ -80,4 +83,19 @@ public SetAnalyzedStatement visitResetStatement(ResetStatement node, Analysis an
         statement.settingsToRemove(settingsToRemove);
         return statement;
     }
+
+    private void checkIfSettingIsRuntime(String name) {
+        checkIfSettingIsRuntime(CrateSettings.CRATE_SETTINGS, name);
+    }
+
+    private void checkIfSettingIsRuntime(List<Setting> settings, String name) {
```

Comment:
is this method called anywhere else than from method above? if not maybe remove this